static/images/Makefile

```diff
@@ -46,3 +46,5 @@ sponsor-arnes.png: Arnes_logo.svg
 logo-atix.png: ATIX-Logo_original.png
 
 logo-blendbyte.png: logo_horizontal.svg
+
+sscg_logo.png: sscg_logo.png
```

Comment:
all the other files are named `logo-$corp` or `sponsor-$corp`, can you please keep that pattern?